### DISCUSSION TITLE
路由后面多了一个斜杠/

### DISCUSSION THREAD

**Discussion by zk-qd** [nuxt/nuxt#9748] [Questions (Nuxt 2)]
Use "nuxt generate" package to generate the project and run it. I found the path followed by a slash, but I don't want this slash
![image](https://user-images.githubusercontent.com/49986675/131801872-7a78a65d-51d2-4fc2-bcf6-b570fcabde43.png)

**Comment by JHerwig** [comment]
If you write in English, more people could help you. I do not even know what language you use.

**Comment by kissu** [comment]
请尝试这个解决方案，也可以用英文写 (try this solution and please write in english)：https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-router#trailingslash

**Reply by zk-qd** [reply]
![image](https://user-images.githubusercontent.com/49986675/131821849-d7d83708-5117-464b-b9be-40c54ddbd668.png)
我之前就尝试过了，但似乎并无作用
![image](https://user-images.githubusercontent.com/49986675/131821945-6f07a2b0-4b34-4ec5-8b49-67e235fc63b7.png)
版本也无问题

**Reply by zk-qd** [reply]
I've tried this before but it doesn't seem to work and the version is fine
